internal/pkg/gateway/registry/registry_test.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2021 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package registry
+
+import (
+	"testing"
+
+	"github.com/hyperledger/fabric/internal/pkg/gateway/mocks"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRegistry_Endorsers(t *testing.T) {
+	endorser := &mocks.Endorser{}
+	registry, err := New(endorser)
+	require.NoError(t, err, "Failed to create registry")
+
+	endorsers := registry.Endorsers("my_channel", "my_chaincode")
+	require.Equal(t, 1, len(endorsers), "There should be one endorser returned")
```

Comment:
There's a matcher for length: https://pkg.go.dev/github.com/stretchr/testify/require#Len